### ISSUE TITLE
Simplify calling (a)sync delayed runnables

### ISSUE THREAD

**Issue by kh498** [inf112-v19/Blodige-roboter#81]
Instead of calling 
```java
GameGraphics.executorService.schedule(() -> Gdx.app.postRunnable(() -> { /* some code */ }), timeInMs,
                      TimeUnit.MILLISECONDS);
```
Something like the code snippet below should be called for synced delayed code (where the synced thread is the application thread)

```java
GameGraphics.scheduleSync(()->{ /* some code */ }), timeInMs);
```

and for async (ran directly on the `executorService` thread)

```java
GameGraphics.scheduleAsync(()->{ /* some code */ }), timeInMs);
```

**Comment by kh498** [comment]
Should also cancel all tasks when changing map

**Comment by kh498** [comment]
Fixed in #108
